examples/seismic/skew_self_adjoint/example_iso.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+from devito import configuration
+from examples.seismic import RickerSource
+from examples.seismic.skew_self_adjoint import *
+
+configuration['language'] = 'openmp'
+configuration['log-level'] = 'DEBUG'
+
+nx, ny, nz = 889, 889, 379
```

Comment:
While I know where this size comes from, this is a bit big for an example.

Reply:
401, 401, 401?

Reply:
I'd got to 201 to be safe but sure.

Reply:
done